CMakeLists.txt

```diff
@@ -39,13 +39,44 @@ set_target_properties(djinni_support_lib PROPERTIES
   CXX_EXTENSIONS false
 )
 
+
+install(
```

Comment:
I think you should add support for `find_package()` being able to find the installed library, and maybe even support library versioning (e.g. `find_package(djinni::supportlib 1.0 REQUIRED`). Have a look at [this article](https://cliutils.gitlab.io/modern-cmake/chapters/install/installing.html) for an example. I don't know about conan but this is a requirement for hunter packages, afaik.

Reply:
A sidenote if I did not clarify this yet: I know that you are planning to bring the support-lib to conan, but I am using hunter and will try to find (or push you in) a way that the lib works with both package-managers. :)

Reply:
hunter and conan are not exclusive, in fact, every package manager on the planet will need the make install step.

adding a cmake file so that find_package works, and therefore also the find_package generator in conan, or hunter, is a nice thing, but is an additional improvement.

Step 1, make it install
Step 2, make it findable

Step 2 will very likely be a new issue and PR, but depends on 1